### PR TITLE
Feature branch

### PR DESCRIPTION
**Pull request by Amtresu**
I've made a completely responsive webpage for an upcoming fictional craft coffee exhibition.
This site includes

- Completely responsive design working on all screen types
- A multi-page layout featuring a home and an about page
- A functioning javascript menu to efficiently navigate between pages
- a functioning home logo button
- a dynamically created "featured roasters" page
- design consistent with the source material
- a descriptive readme giving credit to the source material author
https://www.loom.com/share/5ddbabe2cdfc4c96a24843b0b22f5bce
image
image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,60 @@
...
+- HTML
+- CSS
+- Javascript
+
+## Live Demo 
+
+https://amtresu.github.io/Microverse-Capstone-Project-Module-1/
+ 
+
+## Getting Started
```
Comment (Deepakdanger) [inline README.md:17]: - [ ] Kindly include the code, how to run the project locally.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,60 @@
-# Microverse Capstone Project Module 1
- Here I am making a a website using HTML+CSS(Bootstrap)+ and Javascript
+# Project Name
+
+> This website is to advertise the Global assocation of coffee roasters very first annual coffee exhibition.
```
Comment (Deepakdanger) [inline README.md]: - [ ] Kindly elaborate on the project a little bit more.

**Review by Deepakdanger — CHANGES_REQUESTED** [review]
Hi @Amtresu, 

Good job so far! 👏
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight :100: 
- No linter error :heavy_check_mark: 
- Website looks professional :+1: 

## Required Changes ♻️
- [ ] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [ ] Kindly use the same design as in the original website for the background. This makes website looks good.
- your website
Screenshot from 2022-07-25 15-23-08

- Original website
Screenshot from 2022-07-25 15-22-58

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Amtresu** [comment]
Hi [Deepakdanger](https://github.com/Deepakdanger). thanks for the review. I want to clarify what you highlighted on the original design images. What you instructed me to input is actually part of the image of the source material, and not so much actually in the design if you understand me. I can implement it if necessary, but seeing as we are allowed to change the images I did not incorporate it into mine. Please let me know your thoughts

**Comment by Deepakdanger** [comment]
Hi @Amtresu , Here it's about the background checkers behind the images. You don't have to change the respective images.  just add that background checkers behind to each image.

**Comment by Amtresu** [comment]
Hi [Deepakdanger](https://github.com/Deepakdanger), thanks for replying ,I understand that. What I'm saying is that the checkers on the original site are actually part of the image, and not implemented through coding.

Here is the site we are coding for reference. http://summit.cckorea.org/main.html

**Comment by Deepakdanger** [comment]
ok, @Amtresu, I am putting it as optional, as it makes the website looks good, the remaining changes are mandatory, and complete it before asking for the next review.

**Review by muneebulrehman — APPROVED** [review]
Hi @Amtresu , 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🥳
Congratulations! 🎉 

# Status: Approved✔️

## Highlights\
- No linter errors
- Good follow-up with the previous reviewer.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**